docs/kfdrc_gatk_create_cnv_pon_wf.md

```diff
@@ -7,6 +7,62 @@ a direct port of the GATK best practices workflow WDL. Use this workflow for
 creating a GATK CNV Panel of Normals given a list of normal samples aligned
 reads. Supports both WGS and WES/WXS.
 
+## Panel of Normal Recommendations
+
+### Best Case Scenario
+
+Our Panel of Normal standards are the same as those defined in the Broad/GATK
+Documentation here:
+https://gatk.broadinstitute.org/hc/en-us/articles/360035890631-Panel-of-Normals-PON-.
+
+In the best case scenario, you will have 40 samples derived from healthy
+tissue. Ideally these samples come from young and healthy individuals (thus
+eliminating the chance of a sample containing undiagnosed tumor tissue). Those
+40 samples will then be processed in the same way as the tumor sample.
+Processing includes all technical variables through sequencing (library
+preparation, sequencing platform, etc.). For WGS or WXS that spans the
+allosomes, the samples used for the panel of normals should have uniform sex.
+
+### Imperfect Scenarios
+
+Often when working on a project, there will not be 40 normal samples available
+to the user. If multiple sequencing approaches/centers are used or you have
+allosomal regions of interest, the number of required normals can rapidly
+expand. If you are unable to acquire the 40 samples, it is still possible to
+create a smaller panel of normals. According to GATK, there is no definitive
+rule for the number of samples and even a small panel is better than no panel.
+
+Internally, we have observed that as the panel becomes smaller, more calls are
+made and those calls are less accurate. For WGS, we have observed that once
+panels become smaller than 15 samples, the calls begin to noticeably
+deteriorate. For WXS, calling deteriorates somewhere under 25 samples.
+
+### Run at Own Risk
+
+It is possible to create a panel of normals from a single normal sample.
+Internally, we have observed that the calls generated from this scenario are
+highly erratic. In the case of WXS, the calls made had little overlap with the
+calls made from a larger panel. In the case of WGS, we had instances where the
+results looked slightly worse than the 15 sample panel of normals but we also
+had instances where the calls in no way resembled those from the larger panel.
+Again these results were all better than no panel but were far from what we
+would call reliable calls.
+
+#### Potential Workflow
+
+1. Your tumor sample shares an sequencing approach with 40 normals samples (same sex if the approach includes allosomal regions) in the same project.
```

Comment:
If you wanted to analyze only chr1-22, then the panel can be mixed sex. The panel still needs to be 40 samples to meet the GATK recommendations though.

Reply:
Oh, meaning if you only did CNV on chr 1-22, like if you tried to do chr 1-22, X,Y using a panel that had only chr1-22, would it fail?

Reply:
Last I checked the intervals used to CollectReadCounts needs to be the same across the board. So if you created a PON that was only 1-22 then in the case workflow you did 1-22XY, GermlineCNVCaller would fail with some variety of "intervals do not match"

Reply:
Right, I thought so. Maybe that can be added as guidance, like, if you don't have enough samples to even get 30 per sex, then perhaps alternatively you can create a mixed panel that is allosomal only (chr1-22) **but** you cannot perform CNV analysis on chr X and Y

Reply:
Sure. I can add a little more detail to the mixed sample scenario.

Reply:
Ok. Added. Take a look.